test/acceptance/features/serviceClassPush.feature

```diff
@@ -90,12 +90,14 @@ Feature: Use ServicesClass resources to manage RegisteredService resources (Push
         Then The resource registeredservices.primaza.io/$scenario_id-1:primaza-system is available in cluster "main"
         And jsonpath ".spec.serviceEndpointDefinition[0]" on "registeredservices.primaza.io/$scenario_id-1:primaza-system" in cluster main is "{"name":"host","value":"internal.db.stable.example.com"}"
         And The resource registeredservices.primaza.io/$scenario_id-2:primaza-system is available in cluster "main"
-        And jsonpath ".spec.serviceEndpointDefinition[0]" on "registeredservices.primaza.io/$scenario_id-2:primaza-system" in cluster main is "{"name":"host","value":"external.db.stable.example.com"}"
-        And jsonpath ".metadata.annotations."primaza.io/service-apiversion"" on "registeredservices.primaza.io/$scenario_id-2:primaza-system" in cluster main is "stable.example.com/v1"
-        And jsonpath ".metadata.annotations."primaza.io/service-kind"" on "registeredservices.primaza.io/$scenario_id-2:primaza-system" in cluster main is "Backend"
-        And jsonpath ".metadata.annotations."primaza.io/service-name"" on "registeredservices.primaza.io/$scenario_id-2:primaza-system" in cluster main is "$scenario_id-2"
-        And jsonpath ".metadata.annotations."primaza.io/service-namespace"" on "registeredservices.primaza.io/$scenario_id-2:primaza-system" in cluster main is "services"
-        And jsonpath ".metadata.annotations."primaza.io/cluster-environment"" on "registeredservices.primaza.io/$scenario_id-2:primaza-system" in cluster main is "worker"
+        And On Cluster "main", jsonpaths are satisfied
```

Comment:
Split this into separate assertion steps.  If one of these fail, it's going to be difficult to determine which assertion failed; splitting them makes this debugging process a lot easier.

Reply:
I liked the table 😢 but you're right

Reply:
I liked it too, but I don't think it's going to work for us :(

Reply:
done